style.css

```diff
@@ -177,26 +176,33 @@ html {
   display: flex;
   justify-content: center;
   flex-grow: 1;
+
+  /* Default value, dont need to use it */
   flex-direction: row;
   flex-wrap: nowrap;
   margin: 2rem 0 0.5rem 0;
+
+  /* margins can be overwritten by the parent */
 }
 
+/* Sugestion: insert information in the parent // using clamp instead of calc() */
 .workcard__tags > * {
```

Comment:
Since you only have `<li>` inside your `<ul>`, I don't think you need the `> *` selector 🤔. Kindly consider removing it and see if it doesn't affect your code. That will help it look cleaner